components/elm/src/biogeochem/CNNitrogenStateType.F90

```diff
@@ -967,34 +964,31 @@ subroutine Summary(this, bounds, num_soilc, filter_soilc, num_soilp, filter_soil
    nlev = nlevdecomp
    if (use_pflotran .and. pf_cmode) nlev = nlevdecomp_full
 
-   if (use_nitrif_denitrif .or. (use_pflotran .and. pf_cmode)) then
+   do fc = 1,num_soilc
```

Comment:
I don't think so.  It was an .or..  ANd since use_nitrif_denitrif is implicitly true now, this condition should now always be true, and hence unnecessary.  Right?

Reply:
@rgknox that seems right to me.